### ISSUE TITLE
Simple Analogy for understanding Docker

### ISSUE THREAD

**Issue by brobbins** [dotnet/docs#20557]
Regarding the page for "What is Docker?", I had to read the analogy section a few times before it started to make some sense.  I think the author might have missed a couple of key ideas, so I wanted to share my take on how to word that section so that it could be more easily understand by others. 

Here is my version:

<begin>

Perhaps a simple analogy can help you grasp of the concept of Docker.

Let's go back in time to the 1950's for a moment. There were no word processors, so people used photocopy machines to produce multiple copies of a document as needed.

Imagine you work for a company that has many customers spread out geographically and with different buying patterns.  The Customer Service team needs to send out correspondence letters, but the letters need to be specific to customers in the different purchasing categories.   You are responsible for issuing batches of those letters as required, which will be mailed using real paper and envelopes and delivered physically to each customer's address (no email back then). You are given a few samples of the form letters they have created for this purpose; however, you discover the letters share some common wording.  You realize that they are just compositions of a large set of paragraphs which are picked and arranged as needed, according to the purpose of the letter, so you devise a system to produce the letters quickly (and hopefully get some extra compensation for your effort!)

The system is simple:
1.	You separate the contents of the form letters into distinct paragraphs by putting each one onto a transparent sheet.
2.	To produce one type of letter, you select the paragraphs you need from the collection of sheets, combining them into a custom set, and then align them in the right sequence so they look and read fine.
3.	Finally, you place the set in the photocopier and press the button to produce as many letters as required.  You repeat the process for the next customer category.

Simply put, that is the core idea of Docker. 

<end>

#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3befff27-c4fc-a699-90b3-903ce60234af
* Version Independent ID: 5bed86d7-cb7e-faf5-daf2-f6b03a1dbf66
* Content: [What is Docker?](https://docs.microsoft.com/en-us/dotnet/architecture/microservices/container-docker-introduction/docker-defined)
* Content Source: [docs/architecture/microservices/container-docker-introduction/docker-defined.md](https://github.com/dotnet/docs/blob/master/docs/architecture/microservices/container-docker-introduction/docker-defined.md)
* Product: **dotnet-architecture**
* Technology: **microservices**
* GitHub Login: @nishanil
* Microsoft Alias: **nanil**

**Comment by sughosneo** [comment]
Hello @brobbins, Thank you for your suggestions and feedback. 

Could you please submit a PR to [docs repo](https://github.com/dotnet/docs/pulls) with these changes ? We will do a final review there and merge it with the master branch.

**Comment by sughosneo** [comment]
Hi @brobbins , just checking if you had a chance to look into this further. Thank you.
